lib/main.js

```diff
@@ -87,6 +91,11 @@ export default {
           args.push("-xc++");
           args.push(...atom.config.get("linter-clang.clangDefaultCppFlags").split(/\s+/));
         }
+        if(activeEditor.getGrammar().name === "C++14") {
+          //const language = "c++";
+          args.push("-xc++");
+          args.push(...atom.config.get("linter-clang.clangDefaultCpp14Flags").split(/\s+/));
```

Comment:
You won't have to do this splitting and all if you make this an array, you can just concat then